### PR TITLE
new getter function for nerd db

### PR DESCRIPTION
**Pull request by dirkseidensticker**
ready for testing @nevrome

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: c14bazAAR
 Title: Download and Prepare C14 Dates from Different Source Databases
 Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, emedyd, eubar, euroevol, irdd, jomon, katsianis, kiteeastafrica, medafricarbon, mesorad, pacea, palmisano, radon, radonb.
-Version: 2.0.0
+Version: 2.0.1
```
Comment (nevrome) [inline DESCRIPTION:4]: I think we can probably finish both #127 and #131 and then directly go to 2.1.0. New databases are new features and should therefore probably increment the middle version number.

**Review by nevrome — COMMENTED** [review]
Why are there changes in the binary data/material_thesaurus.rda file, but apparently not in the underlying .csv file? :thinking:

File: data-raw/variable_reference.csv [hunk 2]
```diff
@@ -446,3 +446,22 @@ lon,radonb,LONGITUDE
...
+lat,nerd,Latitude,
+lon,nerd,Longitude,
+comment,nerd,Comments,
+shortref,nerd,Source,
+,nerd,DateID,
+,nerd,OthLabID,
+,nerd,Problems,
+,nerd,Species,
+,nerd,SiteID,
+,nerd,SiteType,
```
Comment (nevrome) [inline data-raw/variable_reference.csv:466]: Our c14bazAAR datatype has a predefined column for sample species and type of site (https://github.com/ropensci/c14bazAAR/blob/master/data-raw/variable_definition.csv). So maybe we could map these variables as well? Or do they have a very different meaning in NERD?

File: DESCRIPTION (hunk 1 above)
Comment (dirkseidensticker) [reply DESCRIPTION:4]: > Why are there changes in the binary data/material_thesaurus.rda file, but apparently not in the underlying .csv file? 🤔

Running `data-raw/data_prep.R` generates a new .rda , does it not?

File: data-raw/variable_reference.csv (hunk 2 above)
Comment (dirkseidensticker) [reply data-raw/variable_reference.csv:466]: Missed those two, thanks for noticing.

**Pull request merged**